src/better_lane_follow/src/LineDetect.cpp

```diff
@@ -0,0 +1,82 @@
+/*
+ * Created by: Raad Khan
+ * Created On: April 23, 2017
+ * Description: Takes in an image feed and generates lane lines.
+ * Usage:
+ * Subscribes to:
+ * Publishes to:
+ */
+
+#include <LineDetect.h>
+#include <opencv2/imgproc/imgproc.hpp>
+#include <opencv2/highgui/highgui.hpp>
+#include <opencv2/objdetect/objdetect.hpp>
+
+LineDetect::LineDetect() : initalLineDetectThreshold(200),
```

Comment:
How would you suggest initialising them? I'm not super particular about it, but I feel like this is a decent format that forces all the variable initialisation to be in one place. Certainly I've seen integers initialised in production codebases like this.